pulsar-client-cpp/CMakeLists.txt

```diff
@@ -212,16 +216,30 @@ include_directories(
   ${GMOCK_INCLUDE_PATH}
 )
 
-set(COMMON_LIBS
-  ${COMMON_LIBS}
-  ${Boost_REGEX_LIBRARY}
-  ${Boost_SYSTEM_LIBRARY}
-  ${CURL_LIBRARIES}
-  ${OPENSSL_LIBRARIES}
-  ${ZLIB_LIBRARIES}
-  ${Protobuf_LITE_LIBRARIES}
-  ${ADDITIONAL_LIBRARIES}
-)
+if (MSVC)
+  set(COMMON_LIBS
+    ${COMMON_LIBS}
+    ${Boost_REGEX_LIBRARY}
+    ${Boost_SYSTEM_LIBRARY}
+    ${Boost_DATE_TIME_LIBRARY}
+    ${CURL_LIBRARIES}
+    ${OPENSSL_LIBRARIES}
+    ${ZLIB_LIBRARIES}
+    ${Protobuf_LITE_LIBRARIES}
+    ${ADDITIONAL_LIBRARIES}
+  )
+else()
+  set(COMMON_LIBS
+    ${COMMON_LIBS}
+    ${Boost_REGEX_LIBRARY}
+    ${Boost_SYSTEM_LIBRARY}
+    ${CURL_LIBRARIES}
+    ${OPENSSL_LIBRARIES}
+    ${ZLIB_LIBRARIES}
+    ${Protobuf_LITE_LIBRARIES}
+    ${ADDITIONAL_LIBRARIES}
+  )
+endif()
```

Comment:
Why not leave the original set non duplicated and add later: 

```
if (MSVC)
  set(COMMON_LIBS
        ${COMMON_LIBS}
        ${Boost_DATE_TIME_LIBRARY}
   )
endif ()	    
```

Reply:
done